### ISSUE TITLE
Desktop Environment not supported. (using Unity7)

### ISSUE THREAD

**Issue by user1007017** [DozyDolphin/Worksets#8]
Hi, 
I'd really like to use your tool!

Unfortunately I'm getting the following: 
```
You seem to be running the Unity:Unity7 desktop environment. Unfortunately, at the moment only the Unity desktop environment is supported.
```

Desktop: Unity:Unity7
Session: ubuntu

Distributor ID:	Ubuntu
Description:	Ubuntu 17.04
Release:	17.04
Codename:	zesty

**Comment by DozyDolphin** [comment]
Hi,
Thank you for your interest. 
I will look into it and get back to you in the following week.

**Comment by DozyDolphin** [comment]
Hi Yoda,

I released [v0.1.1](https://github.com/DozyDolphin/Worksets/releases) which should solve your problem. I've tested it on Ubuntu 17.04 in Virtual Box.

Please let me know if it solves your problem, so I can close this issue.
Also, don't hesitate to bring any feedback my way - good or bad :-)

Have a nice day.
